mti_nma/steps/nma/nma.py

```diff
@@ -37,53 +38,75 @@ class Nma(Step):
 
     def __init__(
         self,
-        clean_before_run=False,
         direct_upstream_tasks=[Avgshape],
-        filepath_columns=["w_FilePath", "v_FilePath", "fig_FilePath"],
-        config=None
+        filepath_columns=["w_FilePath", "v_FilePath", "fig_FilePath"]
     ):
         super().__init__(
-            clean_before_run=clean_before_run,
             direct_upstream_tasks=direct_upstream_tasks,
-            filepath_columns=filepath_columns,
-            config=config
+            filepath_columns=filepath_columns
         )
 
     @log_run_params
-    def run(self, **kwargs):
+    def run(self, mode_list=list(range(6)), **kwargs):
+
+        if platform == "darwin":
+            path_blender = "/Applications/Blender.app/Contents/MacOS/Blender"
+        # elif platform == "linux" or platform == "linux2":
+        #     pass
+        else:
+            raise NotImplementedError(
+                "OSes other than Mac are currently not supported."
+            )
 
         # Load avg shape manifest and read avg mesh file out
         avgshape = Avgshape()
         manifest_filepaths_rel2abs(avgshape)
         df = avgshape.manifest.copy()
```

Comment:
For the future: same here, this info should be parameterized.

Reply:
This will be fixed in the next PR